src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const oldFormat = fromFormat.splice(0, 3);
+  const newFormat = toFormat.splice(0, 3);
+  const rawDate = date.split(oldSeparator);
+
+  let newDate = '';
+
+  for (let j = 0; j < newFormat.length; j++) {
+    for (let i = 0; i < oldFormat.length; i++) {
```

Comment:
loop inside loop 😬
I'm sure that you can better )

Reply:
Thanks:)